### PR TITLE
Figure

### PR DESCRIPTION
**Pull request by Quaquaro**

### CODE REVIEW

**Review by PReichetanz — COMMENTED** [review]
Looks good to me! Just one comment on the main role attribute.

File: index.html [hunk 1]
```diff
@@ -14,10 +14,22 @@ <h1>Christoph Timm</h1>
...
+    <figure>
+        <img src="https://source.unsplash.com/random/600x300" alt="" height="300" width="600">
+        <figcaption>That's me!</figcaption>
+    </figure>
+        
+        <main>
             <article>
                 <h2>Hallo ich bin Christoph Timm</h2>
-                <p>Content 1</p>
+                <p>Cupcake ipsum dolor sit amet cake croissant fruitcake
```
Comment (badabam) [inline index.html:25]: Why is each line in this paragraph more indented?

File: index.html [hunk 2]
```diff
@@ -14,10 +14,22 @@ <h1>Christoph Timm</h1>
                     <a href="#biography">Biography</a>
             </nav>
     </header>
-        <main role="main">
+    <figure>
+        <img src="https://source.unsplash.com/random/600x300" alt="" height="300" width="600">
```
Comment (badabam) [inline index.html:18]: Nice (and correct!) with your `width` and `height`! 👍

**Review by badabam — COMMENTED** [review]
I found a small thing that you could improve in some unrelated branch later. Well done!

**Pull request merged**